barretenberg/cpp/src/barretenberg/commitment_schemes/ipa/ipa.hpp

```diff
@@ -613,12 +584,12 @@ template <typename Curve_> class IPA {
      */
     // TODO(https://github.com/AztecProtocol/barretenberg/issues/912): Return the proper VerifierAccumulator once
     // implemented
-    static VerifierAccumulator reduce_verify(const std::shared_ptr<VK>& vk,
+    static VerifierAccumulator reduce_verify([[maybe_unused]] const std::shared_ptr<VK>& vk,
```

Comment:
Can we simplify this interface or does the vk have to remain to conform to some more general interface?

Reply:
yeah, I thought we had to conform, but I was able to remove it